torch/_dynamo/variables/torch.py

```diff
@@ -699,6 +701,57 @@ def add_subgraph(name, gm):
             tx.output.register_attr_or_module(gm, next_name, source=src)
             return next_name
 
+        def make_subgraph(f, sub_args, graph_checkpoint, checkpoint):
```

Comment:
This should have "speculate" in its name as it doesn't actually apply the changes to Dynamo state